packages/ckeditor5-dev-release-tools/tests/tasks/reassignnpmtags.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { expect } = require( 'chai' );
+const mockery = require( 'mockery' );
+const sinon = require( 'sinon' );
+
+describe( 'reassignNpmTags()', () => {
+	let stubs, reassignNpmTags;
+
+	beforeEach( () => {
+		mockery.enable( {
+			useCleanCache: true,
+			warnOnReplace: false,
+			warnOnUnregistered: false
+		} );
+
+		stubs = {
+			tools: {
+				shExec: sinon.stub()
+			}
+		};
+
+		sinon.stub( console, 'log' );
+		stubs.tools.shExec.withArgs( 'npm whoami' ).resolves( 'authorized-user' );
+		stubs.tools.shExec.withArgs( sinon.match( 'npm show' ) ).resolves( '1.0.0' );
+
+		mockery.registerMock( '@ckeditor/ckeditor5-dev-utils', { tools: stubs.tools } );
+
+		reassignNpmTags = require( '../../lib/tasks/reassignnpmtags' );
+	} );
+
+	afterEach( () => {
+		mockery.deregisterAll();
+		mockery.disable();
+		sinon.restore();
+	} );
+
+	it( 'should throw an error when `authorizedUser` does not match user logged in to npm', async () => {
+		stubs.tools.shExec.withArgs( 'npm whoami' ).resolves( 'incorrect-npm-user' );
+
+		try {
+			await reassignNpmTags( { authorizedUser: 'correct-npm-user' } );
+			throw new Error( 'Expected to throw' );
+		} catch ( e ) {
+			expect( e.message ).to.equal( 'User: incorrect-npm-user is not matching authorized user: correct-npm-user.' );
+		}
+	} );
+
+	it( 'should not update tags when user is not logged in', async () => {
+		stubs.tools.shExec.withArgs( sinon.match( 'npm show' ) ).throws( 'User is not logged in error.' );
```

Comment:
If the `npm show` command throws an error, it does not mean that user is not logged in. The `npm show` command does not require to be logged to npm.